Buildings/Controls/OBC/Utilities/Validation/PIDWithInputGains.mo

```diff
@@ -0,0 +1,100 @@
+within Buildings.Controls.OBC.Utilities.Validation;
+model PIDWithInputGains "Test model for PIDWithInputGains"
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Pulse pulse(
+    period=0.25)
+    "Setpoint"
+    annotation (Placement(transformation(extent={{-30,14},{-10,34}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant const(
+    k=0.5)
+    "Measurement data"
+    annotation (Placement(transformation(extent={{-30,-22},{-10,-2}})));
+  Buildings.Controls.OBC.Utilities.PIDWithInputGains PIDWithInputGains(
+      controllerType=Buildings.Controls.OBC.CDL.Types.SimpleController.PID)
+    annotation (Placement(transformation(extent={{20,-30},{40,-10}})));
+  CDL.Continuous.PIDWithReset PID(
+    controllerType=Buildings.Controls.OBC.CDL.Types.SimpleController.PID,
+    k=1,
+    Ti=0.5,
+    Td=0.1) annotation (Placement(transformation(extent={{20,20},{40,40}})));
+  CDL.Logical.Sources.Pulse resSig(period=1)
+    "Reset signal"
+    annotation (Placement(transformation(extent={{-80,40},{-60,60}})));
+  CDL.Continuous.Sources.Pulse pulse_k(
+    amplitude=0.2,
+    width=0.4,
+    period=1,
+    shift=0.6,
+    offset=1) "k"
+    annotation (Placement(transformation(extent={{-80,0},{-60,20}})));
+  CDL.Continuous.Sources.Pulse pulse_Ti(
+    amplitude=0.2,
+    width=0.4,
+    period=1,
+    shift=0.6,
+    offset=0.5) "k"
+    annotation (Placement(transformation(extent={{-80,-40},{-60,-20}})));
+  CDL.Continuous.Sources.Pulse pulse_Td(
+    amplitude=0.1,
+    width=0.4,
+    period=1,
+    shift=0.6,
+    offset=0.1) "k"
```

Comment:
Review and revise the comment strings. These are important for users to aid in understanding the model.

Reply:
Thanks, I have revised the comment strings.